src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
The stylesheet link references a '.scss' file. Ensure that this SASS file is compiled to CSS and that the correct CSS file is linked here. Browsers cannot directly interpret SASS files.